test/integration/manifests/datapath/linux-deployment-ipv6.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
```

Comment:
can this be added to goldpinger folder to be consistent with existing structure?